portfolio/src/main/webapp/form.html

```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+   <link rel="stylesheet" href="formStyle.css">
+   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">  
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    
+
+</head>
+<body>
+
+<h2>Contact Me</h2>
+
+
+<div id="container">
+  <form action="/form-handler" method = "POST">
+  <div class="row">
+    <div class="col-25">
+      <label for="fname">First Name</label>
```

Comment:
```suggestion
      <label for="firstName">First Name</label>
```

We would rather go for being descriptive than being short, unless the variable name is ridiculously long. 

Please fix for the other instances as well, like "lname".